js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
```

Comment:
@jjeffryes oh I didn't get you, I thought you want to move **everything** to seperate views. Gotcha.